Assets/PatchKit Patcher/Scripts/AppData/Local/TemporaryDirectory.cs

```
@@ -50,57 +44,23 @@ public string GetUniquePath()
             return uniquePath;
```

What should happen if temporary file is not created after 1000 tries? Maybe throw an exception?